Makefile

```diff
@@ -0,0 +1,49 @@
+pip:
+	pip install -e .
+
+pip-development: pip
+	pip install -e.[development]
+
+pip-docs: pip-development
+	pip install -r docs/requirements.txt
+	pip install doc2dash
+
+pip-wheel:
+	pip install wheel
+
+bdist-wheel:
+	python setup.py sdist bdist_wheel
+
+docs:
+	rm -rf docs/pyteal.docset
+	cd docs && \
+	make html && \
+	doc2dash --name pyteal --index-page index.html --online-redirect-url https://pyteal.readthedocs.io/en/ _build/html && \
+	tar -czvf pyteal.docset.tar.gz pyteal.docset
+
+build:
+	python -m scripts.generate_init --check
```

Comment:
This isn't really a build, since it doesn't modify anything. Maybe have two steps, `generate` and `check-generated`?

Reply:
👍